index.html

```diff
@@ -49,55 +54,58 @@ <h3 class="small_text_about_blog">Why Ghost (& Figma) instead of Medium, WordPre
         </section>
         <section class="section_with_photo">
             <h2 class="articles">All articles</h2>
-            <div class="atribute">
-                <img class="second_page_photos" width="304px" height="176px" src="photos\1.jpg"
-                    alt="Granny gives everyone the finger">
-                <div>
-                    <p class="text_after_images">Granny gives everyone the finger, and other tips from OFFF
-                        Barcelona</p>
+            <div class="div_with_photo">
```

Comment:
better not to use tag names in your classes. instead of `div_with_photo` it can just be `photo`